playbooks/02 - Use Case - CICD/Check Playbook is Already Running.json

```diff
@@ -13,41 +13,32 @@
         "playbook_name"
     ],
     "synchronous": false,
-    "collection": "\/api\/3\/workflow_collections\/4c58fed3-9e16-451a-9528-0be2b71acda4",
+    "collection": "\/api\/3\/workflow_collections\/36f63e54-01b1-45f7-bd33-c53dba44b797",
```

Comment:
why is the collection id itself changed? may cause issues during SP upgrade

Reply:
done